prisma/migrations/20210529200211_add_tables_for_nextjs_auth/migration.sql

```diff
@@ -0,0 +1,77 @@
+-- CreateTable
+CREATE TABLE "Account" (
+    "id" TEXT NOT NULL,
```

Comment:
What's the reason for the type being TEXT instead of SERIAL for id? Also, would it be better to just define the primary key here? `id SERIAL PRIMARY KEY NOT NULL`